bundles/verified-customer/src/FondOfKudu/Glue/VerifiedCustomer/Processor/Validator/VerifiedCustomerValidator.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace FondOfKudu\Glue\VerifiedCustomer\Processor\Validator;
+
+use FondOfKudu\Glue\VerifiedCustomer\Dependency\Client\VerifiedCustomerToCustomerInterface;
+use FondOfKudu\Glue\VerifiedCustomer\VerifiedCustomerConfig;
+use Generated\Shared\Transfer\CustomerTransfer;
+use Generated\Shared\Transfer\RestErrorCollectionTransfer;
+use Generated\Shared\Transfer\RestErrorMessageTransfer;
+use Spryker\Glue\GlueApplication\Rest\Request\Data\RestRequestInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class VerifiedCustomerValidator implements VerifiedCustomerValidatorInterface
+{
+    /**
+     * @var \FondOfKudu\Glue\VerifiedCustomer\Dependency\Client\VerifiedCustomerToCustomerInterface
+     */
+    protected VerifiedCustomerToCustomerInterface $customerClient;
+
+    /**
+     * @var \FondOfKudu\Glue\VerifiedCustomer\VerifiedCustomerConfig
+     */
+    protected VerifiedCustomerConfig $config;
+
+    /**
+     * @param \FondOfKudu\Glue\VerifiedCustomer\VerifiedCustomerConfig $config
+     * @param \FondOfKudu\Glue\VerifiedCustomer\Dependency\Client\VerifiedCustomerToCustomerInterface $customerClient
+     */
+    public function __construct(
+        VerifiedCustomerConfig $config,
+        VerifiedCustomerToCustomerInterface $customerClient
+    ) {
+        $this->customerClient = $customerClient;
+        $this->config = $config;
+    }
+
+    /**
+     * @param \Spryker\Glue\GlueApplication\Rest\Request\Data\RestRequestInterface $restRequest
+     *
+     * @return \Generated\Shared\Transfer\RestErrorCollectionTransfer|null
+     */
+    public function isVerified(RestRequestInterface $restRequest): ?RestErrorCollectionTransfer
+    {
+        if (!in_array($restRequest->getResource()->getType(), $this->config->getResourcesToBlock())) {
```

Comment:
Ich fände den Weg über eine Whitelist besser oder müssten wir dann zu viel eintragen? 🤔

Reply:
Good point.. will check what we can do instead